web.config.js

```diff
@@ -42,6 +42,36 @@ function createAppSettingsEnv(env, isXMLTransform = false) {
 	`
 }
 
+/** @param {import('.').createWebConfigOptions["iisNodeOptions"]} options */
+function createIISNodeConfig(options) {
+  const defaults = {
+    enableXFF: true,
+    nodeProcessCommandLine: 'node.exe',
+    watchedFiles: 'web.config;node_modules*;*.js;*.cjs',
+  }
+
+  let attributes = Object.entries(options).reduce((acc, [key, value]) => {
```

Comment:
nice reducer :)

Reply:
There seems to be a weird bug with reduce so I am gonna have to take it out